ci/Dockerfile

```diff
@@ -5,7 +5,7 @@ RUN apk update && apk add --no-cache curl
 # curl kubectl instead of ADD'ing it, since we want to cache the layer
 RUN curl -sfL \
       -o /usr/local/bin/kubectl \
-      https://storage.googleapis.com/kubernetes-release/release/v1.8.2/bin/linux/amd64/kubectl && \
+      https://storage.googleapis.com/kubernetes-release/release/v1.9.2/bin/linux/amd64/kubectl && \
```

Comment:
I don't recall where this is used... why can't we default to the packages burned in by wardroom?

/cc @chuckha

Reply:
This is used by CI. 

CI build this docker image and then executes tests from within this image.

This allows our CI environment to not need kubectl to run and only need docker.
I'm happy reevaluating this